### PR TITLE
[fix bug 1288802] Misplaced down arrow, in select initiave dropbox

### PR DESCRIPTION
**Pull request by tsmrachel**
https://bugzilla.mozilla.org/show_bug.cgi?id=1288802

### CODE REVIEW

File: remo/base/static/base/css/app-fd4.less [hunk 1]
```diff
@@ -2135,32 +2134,42 @@ meter::-o-meter-suboptimum-value {
...
+    }
+
+    .required-field-input:before {
+         &:extend(.required-field:before);
+    }
+
+    .required-field-input {
+        &:extend(.required-field);
+        left: 5px;
+        top: 10px;
```
Comment (comzeradd) [inline remo/base/static/base/css/app-fd4.less]: Same here: try `top: 18px`.

File: remo/base/static/base/css/app-fd4.less [hunk 2]
```diff
@@ -2135,32 +2134,42 @@ meter::-o-meter-suboptimum-value {
...
+
+    .section {
+        padding-right: 25px;
+        padding-bottom: 25px;
+    }
+
+    .required-field {
+        &:extend(.required-field);
+        left: 5px;
+        top:0;
```
Comment (comzeradd) [inline remo/base/static/base/css/app-fd4.less]: Minor tweak: try this to be `top: 10px`, so it's vertical aligned to the middle of the input field.

Comment (tsmrachel) [inline remo/base/static/base/css/app-fd4.less]: if it's to be vertical aligned to the middle, do you mean

.required-field-input top: 20px
.required-field top: 18px

_top: 20px_
input-20

_top: 10px_
input-10

**Pull request merged**